doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -16,7 +20,7 @@ enable more features, such as distributed memory execution.
 0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system
    tools and packages you may need to have installed to build and run Chapel.
 
-1) If you don't already have Chapel 1.15, see
+1) If you don't already have Chapel 1.16, see
```

Comment:
Note from my earlier [comment](https://github.com/chapel-lang/chapel/issues/5963#issuecomment-292365470) that these should stay pointing towards 1.15 for now, since 1.16 tarball does not exist.